sdk/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -241,6 +241,9 @@ const (
 	// for the request. This data is synthetic; it is not stored in etcd and instead is only applied when retrieving
 	// CRs for the CRD.
 	AnnotationAPIIdentityKey = "apis.kcp.io/identity"
+	// AnnotationKcpValidateNameKey is the annotation key used to indicate that a CRD should be validated
+	// not as the default DNS subdomain.
+	AnnotationKcpValidateNameKey = "internal.kcp.io/validate-name"
```

Comment:
this is not really API. Wouldn't expect this here. Can't we just reference the constant from the kube fork?